### ISSUE TITLE
Time checker Regex on 80

### ISSUE THREAD

**Issue by antsankov** [ridler/myCal#2]
In line 80 you check if the format of the time is:
`var timeFormat = /[0-9][0-9]:[0-9][0-9]/;`

This is fine, but what if the user types `99:99`. This would still be invalid, correct? 

Just pointing it out because this has to potential to introduce garbage into your DB which can percolate into bad aggregate results. 

Otherwise the code looks great!

**Comment by ridler** [comment]
Good catch!  I will definitely fix this if we go further with the assignment.  Thanks for reviewing! :smile:
